src/utils/v2/fundingCycle.ts

```diff
@@ -55,19 +60,38 @@ const bits1 = 0b1
 const bigNumberToBoolean = (val: BigNumber) => Boolean(val.toNumber())
 
 const parameters: {
-  name: keyof V2FundingCycleMetadata
-  bits: 0 | 1 | 8 | 16
-  parser?: (val: BigNumber) => string | boolean | BigNumber | number | undefined
+  name?: keyof V2FundingCycleMetadata
+  bits: 0 | 1 | 2 | 8 | 16
+  parser?: (
+    val: BigNumber,
+  ) =>
+    | string
+    | boolean
+    | BigNumber
+    | number
+    | undefined
+    | V2FundingCycleMetadataGlobal
 }[] = [
   {
     name: 'version',
     bits: 8,
     parser: (val: BigNumber) =>
       val.toNumber() as V2FundingCycleMetadata['version'],
   },
+  {
+    name: 'global',
+    bits: 16,
+    parser: val => {
+      return {
+        allowSetTerminals: bigNumberToBoolean(BigNumber.from(val).shr(1)),
+        allowSetController: bigNumberToBoolean(BigNumber.from(val).shr(2)),
+      }
+    },
+  },
   { name: 'reservedRate', bits: 16 },
   { name: 'redemptionRate', bits: 16, parser: invertPermyriad },
   { name: 'ballotRedemptionRate', bits: 16, parser: invertPermyriad },
+  { bits: 2 }, // 2 "spacer" bits
```

Comment:
There's a "gap" in the metadata of 2 bits: ballotRedemptionRate ends on bit 71, bits 72 and 73 are empty, then pausePay starts on bit 74.

Might be a bug in the contracts, not sure. Asked in Discord. Inconsequential for us anyway.

Reply:
Ok cool 👍